db/seeds.rb

```diff
@@ -6,32 +6,123 @@
 
 # Add some (unset) settings, to prompt people to set them
 # TODO: replace this with the half-planned site/admin/user overrideable thing
-seed Setting, { name: I18n.t( 'settings.admin_ip_list' ) }, {
+seed Setting, { name: I18n.t( 'setting.admin_ip_list' ) }, {
   value: '',
   description: 'Comma/space-separated list of IP addresses allowed to access admin area'
 }
-seed Setting, { name: I18n.t( 'settings.default_page' ) }, {
+seed Setting, { name: I18n.t( 'setting.default_page' ) }, {
   value: '',
   description: 'Default top-level page (either its name or its slug)'
 }
-seed Setting, { name: I18n.t( 'settings.default_section' ) }, {
+seed Setting, { name: I18n.t( 'setting.default_section' ) }, {
   value: '',
   description: 'Default top-level section (either its name or its slug)'
 }
 
-# Feature flags
-seed FeatureFlag, { name: I18n.t( 'admin.features.user_login' ) }, {
+# Feature Flags (to turn on/off areas of site functionality)
+seed FeatureFlag, { name: I18n.t( 'feature.user_login' ) }, {
   description: 'Allow users to log in',
-  enabled: true,
+  enabled: false,
   enabled_for_admins: true,
 }
-seed FeatureFlag, { name: I18n.t( 'admin.features.user_profiles' ) }, {
+seed FeatureFlag, { name: I18n.t( 'feature.user_profiles' ) }, {
   description: 'Allow viewing of user profiles',
   enabled: true,
   enabled_for_admins: true,
 }
-seed FeatureFlag, { name: I18n.t( 'admin.features.user_registration' ) }, {
+seed FeatureFlag, { name: I18n.t( 'feature.user_registration' ) }, {
   description: 'Allow users to create accounts',
   enabled: true,
   enabled_for_admins: true,
 }
+
+# Capabilities (for user authorisation)
+general_cc = seed CapabilityCategory, { name: 'capability.general'        }
+pages_cc   = seed CapabilityCategory, { name: 'capability.pages'          }
+shared_cc  = seed CapabilityCategory, { name: 'capability.shared_content' }
+users_cc   = seed CapabilityCategory, { name: 'capability.users'          }
+# General
+seed Capability, { name: I18n.t( 'capability.view_admin_area'    ) }, {
```

Comment:
I'm wondering if there is a risk here that dodgy translations (maliciously or accidentally introduced) could workaround authorisation checks.

Similarly, is there potential for enabling/disabling multiple features if duplicate translations are used? 

If the translations are edited during a site's lifetime - would this require migrating the db records relying on those translations?

If any of the above are possible, consider not translating these aspects of the app.